Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/VisualThemes/ThemeEngines/InteractableColorTheme.cs

```diff
@@ -1,6 +1,7 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
```

Comment:
I am a little thrown off by how the InteractableColorTheme is setup. The fact that it inherits from InteractableShaderTheme is only helpful when the target is a Renderer. For other cases where the theme is attempting to change the text color, it does not leverage the InteractableShaderTheme. Can we split this into 2 classes - InteractableColorTheme - addressing Renderers, and InteractableFontColorTheme  - addressing teh text properties? 
(You can choose to Won't Fix it for this PR, but i would really like this to get addressed at some point)

Reply:
yea....you and me both. 

The problem is InteractableColorTheme is one of the more common themes. I mean the addition of the text types targeting was added based on user feedback of people wanting to use this theme. 

The problem with creating the new InteractableFontColorTheme is then existing users will need to migrate because their existing set-ups won't work anymore. And that's not something I can necessarily know ahead of time.

This is something I left more as-is because I'm already rocking the boat quite a lot with this refactor and this just seemed lower priority

I created #5971 to track this issue because yes I won't fix for now. I don't want to add more risk for this refactor that has already taken quite a lot of time